windows_containers/windows-node-upgrades.adoc

```diff
@@ -0,0 +1,15 @@
+[id="windows-node-upgrades"]
+= Windows node upgrades
+include::modules/common-attributes.adoc[]
+:context: windows-node-upgrades
+
+toc::[]
+
+:FeatureName: Windows containers
+include::modules/technology-preview.adoc[leveloffset=+0]
+
+You can ensure your Windows nodes have the latest updates by upgrading the Windows Machine Config Operator (WMCO).
```

Comment:
We should call out that updates are only for k8s components and we don't upgrade the OS.

Reply:
I've included that info as an **Important** admonition in the included module proceeding that line: 

>The WMCO is not responsible for Windows operating system updates. You provide the Windows image when creating the VMs; therefore, you are responsible for providing an updated image. You can provide an updated Windows image by changing the image configuration in the MachineSet spec.

I'll add, specifically, it's only responsible for updating Kubernetes components :+1: